spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -91,6 +92,9 @@ class ShellWidget(NamepaceBrowserWidget, HelpWidget, DebuggingWidget,
     sig_kernel_started = Signal()
     sig_kernel_reset = Signal()
 
+    # Request configuration from plugins
+    sig_configure_requested = Signal()
```

Comment:
```suggestion
    # Request plugins to send additional configuration to the kernel
    sig_config_kernel_requested = Signal()
```

This follows my suggestion above.